packages/data-stores/src/site/constants.ts

```diff
@@ -1,3 +1,8 @@
+import { isEnabled } from '@automattic/calypso-config';
+
 export const STORE_KEY = 'automattic/site';
 
-export const PLACEHOLDER_SITE_ID = 224076220; // creatio2demo.wordpress.com
+export const getPatternSourceSiteID = () =>
```

Comment:
How can we avoid having `getPatternSourceSiteID` here and also in "pattern-assembler/constants.ts"?

Reply:
Oh, I see now that they are different functions.

Reply:
Oops, I missed this haha. Yeah, we need to create two functions!

Reply:
I turned the constant to a function named `getPlaceholderSiteID` and checked that it's only been used in the assembler. See https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso%20PLACEHOLDER_SITE_ID&type=code